frontend/src/ui-components/AcmTable/AcmTable.tsx

```diff
@@ -1073,8 +1075,8 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
                 >
                     {props.emptyState ?? (
                         <AcmEmptyState
-                            title={`No ${props.plural} found`}
-                            message={`You do not have any ${props.plural} yet.`}
+                            title={t('No {{items}} found', { items: props.plural })}
+                            message={t('You do not have any {{items}} yet', { items: props.plural })}
```

Comment:
I'm not sure if this will work - we should ask the translation team if a double translation like this is OK. Some languages might not use the plural form of the word when substituting in. We might have to require that tables either have an empty state or have 2 properties to specify these strings that are used for the default empty state.

Reply:
Will first make initial inquiry about the double translation to translation team.

Reply:
reached out to translation team 11/7/22.  Waiting for feedback

Reply:
@KevinFCormier understood.
I'd like to double confirm that when translating the strings, if there's a segment containing possible plural form, for example "selected %s node", will there be a "selected %s nodes" for the plural candidate?

Reply:
Hey @rh-tchuang,

Yes, with i18next, when we have an interpolation in the message with the name `count`, we automatically get a 2nd key with the suffix `_plural`. See https://github.com/stolostron/console/blob/356ab3059f1b222fee26ab8f7ee83cceb2740f31/frontend/public/locales/en/translation.json#L19:L20 for example.

The translation library automatically selects the right one at runtime based on the value of `count`. Note that this format for plurals is outdated (format v3), but for compatibility reasons with OpenShift console, we will be maintaining it for awhile. Eventually we will need to migrate to the updated format, which uses different suffixes like `_one` or `_other`. https://www.i18next.com/translation-function/plurals 

But it should be clear from the English string whether it is a plural or singular case.

The example in this PR was not really about plurals. 
`message={t('You do not have any {{items}} yet', { items: props.plural })}`
Here, `props.plural` contains a translated term for the type of item we are displaying. For example, for a table of applications, `props.plural` will be "applications". If there are no applications, the message "You do not have any applications yet" will be displayed. This means we are inserting a translated word into another translated message, which I imagine is not ideal. There is no singular form for this case.